Bottom: e3cf38bdcad51990bb50095e81e82cc476f98bb8
Top:    8aa43fb6a71227e4c5ceae602f91c240f3e6e930
Author: Pavel Roskin <proski@gnu.org>
Date:   2013-11-06 23:54:50 -0500

Refresh of mirror

---

diff --git a/farm-checkout b/farm-checkout
index dda81b5..fa15afb 100755
--- a/farm-checkout
+++ b/farm-checkout
@@ -26,7 +26,7 @@ for tag in $tags; do
 	   git clone --quiet --local --mirror $linuxgit $gittree &&
 	   git --bare --git-dir=$gittree config core.bare false &&
 	   git --bare --git-dir=$gittree config core.worktree $srctree &&
-	   git --bare --git-dir=$gittree checkout --quiet $tag &&
+	   git --bare --git-dir=$gittree --work-tree=$srctree checkout --quiet $tag &&
 	   echo "gitdir: $gittree" >$srctree/.git; then
 		echo -e "\e[32mOK\e[0m"
 	else
